src/compiler/transformers/decorators-to-static/convert-decorators.ts

```diff
@@ -105,8 +120,9 @@ const removeStencilDecorators = (classMembers: ts.ClassElement[]) => {
           m.body
         );
       } else if (ts.isPropertyDeclaration(m)) {
-        return ts.updateProperty(m, newDecorators, m.modifiers, m.name, m.questionToken, m.type, m.initializer);
+        return m;
```

Comment:
IIUC, we're not removing Stencil-decorators on the class member anymore if it's a property declaration and deferring that to later - is that correct? If that's the case, it may be worth adding a comment stating why we are singling out property declarations to only return them